arborist/server_test.go

```diff
@@ -2643,6 +2649,15 @@ func TestServer(t *testing.T) {
 			})
 		})
 
+		t.Run("RevokePolicyThroughUser", func(t *testing.T) {
```

Comment:
```suggestion
		t.Run("RevokeGroupPolicyThroughUser", func(t *testing.T) {
```

Reply:
nvm ^
i just saw that the test name is displayed as `Group/RevokePolicyThroughUser`